### PR TITLE
Fixes BinaryCompatBaseline for AndroidXMigration

### PR DESCRIPTION
**Pull request by josegallardo**

### CODE REVIEW

**Comment by jtorres** [comment]
merge when https://github.com/xamarin/md-addins/pull/5957 is fixed & ready

File: main/build/MacOSX/BinaryCheckerConfig.txt [hunk 1]
```diff
@@ -13,5 +13,6 @@
 !Contents/Resources/lib/monodevelop/AddIns/Xamarin.Interactive.XS/Xamarin Inspector.app
 !Contents/Resources/lib/monodevelop/bin/Microsoft.VisualStudio.Imaging.dll
 !Contents/Resources/lib/monodevelop/bin/ServiceHub
+!Contents/Resources/lib/monodevelop/AddIns/MonoDevelop.MonoDroid/Microsoft.CodeAnalysis.Workspaces.MSBuild.dll
```
Comment (Therzok) [inline main/build/MacOSX/BinaryCheckerConfig.txt]: We should most likely be shipping this from MonoDevelop.Core.

**Comment by Therzok** [comment]
Alternative PR: https://github.com/mono/monodevelop/pull/9349

**Comment by Therzok** [comment]
Alternate approach: https://github.com/mono/monodevelop/pull/9349

**Pull request closed without merging**